src/handlers/motions/motionStaked/motionStaked.ts

```diff
@@ -39,6 +40,7 @@ export default async (event: ContractEvent): Promise<void> => {
       motionData: { usersStakes },
       motionData,
     } = stakedMotion;
+    motionData.events.push({ name: 'MotionStaked', initiatorAddress: staker, vote: BigNumber.from(vote).toString(), amount: BigNumber.from(amount).toString() });
```

Comment:
`events` can be destructured like usersStakes

Reply:
vote and amount are already BigNumbers because they're typed as `uint256`, so you don't need to create them using `BigNumber.from`

Reply:
And for consistency, can we write this as:

``` js

const updatedEvents = [
 ...events,
{     
      name: 'MotionStaked',
      initiatorAddress: staker,
      vote:  vote.toString(),
      amount: amount.toString()
}
]

...

await updateMotionInDB(id, {
      ...motionData,
      usersStakes: updatedUserStakes,
      motionStakes,
      remainingStakes,
      events: updatedEvents,
    });

```